autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,6 +1,106 @@
+import asyncio
+import time
+from typing import List
 from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Course, Instructor, Section, Meeting
+from scraper.models.course import generate_course_id
+from scraper.models.section import generate_meeting_id, generate_meeting_time
+
+def parse_section(courseData,instructor_object):
+    #gets values to create object
+    section_id = int(courseData['id'])
+    section_subject = courseData['subject']
+    course_number = courseData['courseNumber']
+    section_number = int(courseData['sequenceNumber'])
+    section_term_code = courseData['term']
+    section_min_credits = courseData['creditHourLow']
+    section_max_credits = courseData['creditHourHigh']
+    section_max_enrollment = courseData['maximumEnrollment']
+    section_current_enrollment = courseData['enrollment']
+    section_instructor = instructor_object
+    #creates and saves section object
+    section_object = Section(id=section_id, subject=section_subject, course_num=course_number, section_num=section_number, term_code=section_term_code, min_credits=section_min_credits, max_credits=section_max_credits, max_enrollment=section_max_enrollment, current_enrollment=section_current_enrollment, instructor=section_instructor)
+    section_object.save()
+    #calls parse meeting for each meeting time in the section
+    meeting_count=0
+    for i in courseData['meetingsFaculty']:
+        parse_meeting(i,section_object,meeting_count)
+        meeting_count+=1
+
+def parse_meeting(meetingsData,section_object,meeting_count):
```

Comment:
```suggestion
def parse_meeting(meetings_data,section_object,meeting_count):
```

Python uses snake style naming conventions rather than camel case. Pylint should find these errors.